
------------------------------
Fecha y hora:
21/11/2021 16:49:37
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
21/11/2021 16:49:38
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
21/11/2021 16:49:38
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
21/11/2021 16:50:00
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
21/11/2021 16:50:29
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Generics\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
21/11/2021 16:51:14
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\TP-04\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
25/11/2021 14:58:12
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
25/11/2021 14:58:16
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
25/11/2021 14:59:55
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
25/11/2021 14:59:55
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
25/11/2021 14:59:59
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
25/11/2021 15:00:07
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
25/11/2021 15:00:08
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
25/11/2021 15:00:08
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
25/11/2021 15:01:36
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
25/11/2021 15:01:36
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
25/11/2021 15:01:36
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
25/11/2021 15:01:56
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
25/11/2021 15:01:56
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
25/11/2021 15:01:56
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
25/11/2021 15:02:14
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 136

------------------------------
Fecha y hora:
25/11/2021 15:02:14
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 196

------------------------------
Fecha y hora:
25/11/2021 15:02:15
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 315

------------------------------
Fecha y hora:
25/11/2021 16:11:36
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137

------------------------------
Fecha y hora:
25/11/2021 16:11:37
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197

------------------------------
Fecha y hora:
25/11/2021 16:11:37
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316

------------------------------
Fecha y hora:
25/11/2021 16:51:07
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137

------------------------------
Fecha y hora:
25/11/2021 16:51:07
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197

------------------------------
Fecha y hora:
25/11/2021 16:51:07
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316

------------------------------
Fecha y hora:
25/11/2021 16:52:00
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137

------------------------------
Fecha y hora:
25/11/2021 16:52:00
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197

------------------------------
Fecha y hora:
25/11/2021 16:52:00
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316

------------------------------
Fecha y hora:
25/11/2021 16:54:55
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137

------------------------------
Fecha y hora:
25/11/2021 16:54:55
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197

------------------------------
Fecha y hora:
25/11/2021 16:54:55
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316

------------------------------
Fecha y hora:
25/11/2021 17:05:43
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 137

------------------------------
Fecha y hora:
25/11/2021 17:05:43
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 197

------------------------------
Fecha y hora:
25/11/2021 17:05:43
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 316

------------------------------
Fecha y hora:
27/11/2021 01:27:54
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 143
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 143
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 143

------------------------------
Fecha y hora:
27/11/2021 01:27:54
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 203
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 203
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 203

------------------------------
Fecha y hora:
27/11/2021 01:27:55
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 322
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 322
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 322

------------------------------
Fecha y hora:
27/11/2021 01:28:28
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 143
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 143
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 63
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 143

------------------------------
Fecha y hora:
27/11/2021 01:28:28
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 203
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 203
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 59
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 203

------------------------------
Fecha y hora:
27/11/2021 01:28:28
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 322
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 322
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\Lautaro\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 322

------------------------------
Fecha y hora:
29/11/2021 15:15:50
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 139
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la lista ya se encuentra llena.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 139
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 139

------------------------------
Fecha y hora:
29/11/2021 15:15:50
Tipo de exepcion:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 199
Error:
EntidadesHostel.PersonaException: No es posible agregar la persona porque la persona ya se encuentra en la lista.
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 199
stack trace:
   at EntidadesHostel.Personas`1.op_Addition(Personas`1 personas, T personita) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Personas.cs:line 58
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 199

------------------------------
Fecha y hora:
29/11/2021 15:15:50
Tipo de exepcion:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 318
Error:
System.IO.FileNotFoundException: El archivo no fue encontrado.
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 318
stack trace:
   at EntidadesHostel.Xml`1.Deserializar(String ruta) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\EntidadesHostel\Serializacion\Xml.cs:line 62
   at TestConsolaHostel.Program.Main(String[] args) in D:\Desktop\test recuperatorios tps\tps_laboratorio_ii\RecuperatoriosTP\TP4\TestConsolaHostel\Program.cs:line 318
